UnitystationLauncher/Views/MainWindow.xaml.cs

```diff
@@ -46,9 +49,6 @@ public MainWindow()
                 _maximizeeButton = this.FindControl<Button>("MaximizeeButton");
                 _closeButton = this.FindControl<Button>("CloseButton");
```

Comment:
Is it not possible to also do this with bindings?
Also `Maximizee` has one e too many.

Reply:
> Is it not possible to also do this with bindings?
> Also `Maximizee` has one e too many.

I don't know way with command bindings for solve this.